### PR TITLE
Update Readme.md from Andrew Klmn with text and picture

### PR DESCRIPTION
**Pull request by andrewklmn**
## What?
I have added this pull request just for learning purposes.

## Why?
I going to become a real Front End Developer

## How?
I have added some text and one picture of my dog Dyuka. 

## Testing?
There is no test because it has no code for testing.

## Anything Else?
Thanks to Kottans for this learning possibility!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1361,3 +1361,17 @@ I'm [Vlad](https://github.com/5Mountains/kottans-frontend), Front-end developer
...
+
+
+# Hello there!
+
+
+My name is [Andriy](https://github.com/andrewklmn/) and I want to be a real Front End Developer.
+
+Here are so many different cats in this Readme.md. Of course, it is pretty much not so bad. But I want to put here something actually better.  Here it is:
+
+![mr Dyuka](https://raw.githubusercontent.com/andrewklmn/freecodecamp/master/images/dyuka.jpg)
```
Comment (zonzujiro) [inline README.md:1375]: (whispering) I am also think that dogs better than cats

Comment (AMashoshyna) [reply README.md:1375]: @zonzujiro you are aware that this is a public repo, right?

Comment (zonzujiro) [reply README.md:1375]: I have no idea what you talking about 😂

**Review by Temu4 — CHANGES_REQUESTED** [review]
Good job! 👍 
But can you get the latest updates form origin repo and resolve conflicts?

**Comment by andrewklmn** [comment]
> Good job! +1
> But can you get the latest updates form origin repo and resolve conflicts?

I got updates to my fork repo and resolved it.
## This branch has no conflicts with the base branch

It seems like there is no conflict now... 
And what I have to do next? 
Could you merge it now?

**Pull request merged**